packages/cli/commands/project/open.js

```diff
@@ -0,0 +1,70 @@
+const open = require('open');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+  addTestingOptions,
+} = require('../../lib/commonOpts');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  getProjectConfig,
+  getProjectDetailUrl,
+  verifyProjectExists,
+} = require('../../lib/projects');
+const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.open';
+
+exports.command = 'open';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  await loadAndValidateOptions(options);
+
+  const accountId = getAccountId(options);
+  const { project } = options;
+  const { projectConfig } = await getProjectConfig();
+  let projectName = project;
+  if (projectName) {
```

Comment:
If a `--project` option is provided, check if project exists based on the default account, or a provided account using the `--account` option. This can be used in any directory with a hubspot.config.yml file.